tests/api_test.go

```diff
@@ -19,6 +19,7 @@ import (
 	"gotest.tools/assert/cmp"
 
 	"agola.io/agola/internal/services/config"
+	"agola.io/agola/internal/services/configstore/common"
```

Comment:
Since this is an integration test it's better to not import internal services types. Just define them in tests/setup_test.go